src/SourceBuild/patches/roslyn/0004-Enable-tests-in-VMR.patch

```diff
@@ -0,0 +1,43 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "Nikola Milosavljevic (CLR) false" <nikolam@microsoft.com>
+Date: Tue, 11 Feb 2025 23:44:46 -0800
+Subject: [PATCH] Enable tests in VMR
+
+Backport: https://github.com/dotnet/roslyn/issues/76960
```

Comment:
Any reason why this isn't already backported as a PR?

Reply:
Didn't get time to complete it yesterday. Will do it right now.

Reply:
Created a backport PR (https://github.com/dotnet/roslyn/pull/77179) and updated the patch with new backport link.